HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -84,6 +84,9 @@ def self.apply(runner, hpxml, hpxml_bldg, eri_version, weather, epw_file: nil, s
     # Do HVAC sizing after all other defaults have been applied
     apply_hvac_sizing(runner, hpxml_bldg, weather, output_format, design_load_details_output_file_path)
 
+    # Set infiltration values for energy simulation after sizing
+    apply_leakiness_description(hpxml_bldg)
```

Comment:
This is not because it needs information from hvac sizing, but I do want hvac sizing uses only the leakiness description information when only leakiness description is provided. And whenever both leakiness and ach/cfm are provided by user, process ach/cfm as a higher priority. So it ends in this way, probably not the best way so let me know if you have thoughts on how to improve.

Reply:
Oh wow, interesting issue. I don't think we've ever run into this kind of situation before. I'll give it some thought...

Reply:
Just documenting our solution here: we decided to introduce a new optional ManualJ input to specify which infiltration method to use.